maps_dashboards/public/components/layer_control_panel/layer_control_panel.tsx

```diff
@@ -315,6 +326,18 @@ export const LayerControlPanel = memo(
       );
     }
 
+    const getLayerTooltipContent = (layer: MapLayerSpecification) => {
+      if (zoom < layer.zoomRange[0] || zoom > layer.zoomRange[1]) {
+        return `Layer is not visible outside of zoom range ${layer.zoomRange[0]} - ${layer.zoomRange[1]}`;
+      } else {
+        return `Layer is visible within zoom range ${layer.zoomRange[0]} - ${layer.zoomRange[1]}`;
```

Comment:
This single message can cover both cases?
`This layer is visible only with zoom range ${layer.zoomRange[0]} - ${layer.zoomRange[1]}`;

Reply:
Good catch, currently we will only need tooltip for invisible layers, will add more content for visible layers.